marimo/_save/save.py

```diff
@@ -206,9 +211,41 @@ def loader(self) -> Loader:
         assert self._loader is not None, UNEXPECTED_FAILURE_BOILERPLATE
         return self._loader()
 
+    def __get__(
+        self, instance: Any, _owner: Optional[type] = None
+    ) -> _cache_call:
```

Comment:
Since this is somewhat complex logic, can you leave a 1 or 2 sentence docstring for posterity? :)